lib/assets/javascripts/twine.coffee

```diff
@@ -124,6 +121,18 @@
 
     Twine
 
+  # IE and Safari don't support node.children for DocumentFragment and SVGElement nodes.
+  # If the element supports children we continue to traverse the children, otherwise
+  # we stop traversing that subtree.
+  # https://developer.mozilla.org/en-US/docs/Web/API/ParentNode.children
+  # As a result, Twine are unsupported within DocumentFragment and SVGElement nodes.
+  #
+  # We also prevent nodes from being iterated over more than once by locking
+  # the children, which prevents nodes that are dynamically inserted as siblings
+  # from causing double/ missed binds and unbinds.
```

Comment:
Do you know if any functionality is depending on this? If we're currently double binding, is it possible people have put workarounds into their code to, for example, remove one bind?

Reply:
Nothing during my general tophatting appeared broken, I can't guarantee that it does not but I think it's just as likely something is silently broken by this (example: you *remove* a sibling element when during a bind on the second last element in the list, now you have already iterated over the current size of the list so the binding stops, even though the last item was never traversed)